### PR TITLE
Us12 implement mongo db

### PR DESCRIPTION
**Pull request by BirkBee**
Changed backend method and implemented MongoDB.

### CODE REVIEW

File: lib/useFavorites.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  function toggleFavorite(id) {
+    if (favorites.includes(id)) {
+      setFavorites(favorites?.filter((favorite) => favorite !== id));
+    } else {
+      setFavorites([...favorites, id]);
+    }
+  }
+
+  return { toggleFavorite, favorites };
+}
```
Comment (Roland-Hufnagel) [inline lib/useFavorites.js:16]: Nice custom hook 💪 Good idea to save favourites local since you don't have a user login.

File: pages/[id].js [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+  if (isLoading) {
     return <p>Loading...</p>;
   }
 
+  async function handleDeleteOutivity() {
+    const response = await fetch(`/api/outivities/${id}`, { method: "DELETE" });
+
+    if (!response.ok) {
+      console.log(response.status);
+      return;
```
Comment (Roland-Hufnagel) [inline pages/[id].js:24]: What if the response.ok is false? You make a console.log but the user is neither informed nor pushed to the homepage.

**Review by Roland-Hufnagel — APPROVED** [review]
What a great job🎉! Really a fine implementation. I left two comments you may want to look at.

File: pages/[id].js (hunk 2 above)
Comment (Julia-Pickel) [reply pages/[id].js:24]: Thanks for taking your time to review our code @Roland-Hufnagel 🙌 

We made some updates here, thanks for the hint!

**Pull request merged**